src/plugins/ui_actions/public/service/ui_actions_service.ts

```diff
@@ -55,6 +55,10 @@ export class UiActionsService {
     this.triggerToActions.set(trigger.id, []);
   };
 
+  public readonly hasTrigger = (triggerId: string): boolean => {
+    return Boolean(this.triggers.get(triggerId));
+  };
```

Comment:
Rather than swallowing the error that is thrown when trying to attach an action to a trigger that doesn't exist, I wanted a way to ensure that the trigger exists **before** attaching the action - this is useful for situations like the new Canvas add panel trigger, which is not registered in Serverless (for example) and thus was throwing an error. Refer to https://github.com/elastic/kibana/pull/179667/files#diff-65c205deed13f182061f156d76c8108c8b68a0d51d27100e36e22263ca087b3e.